### ISSUE TITLE
Issue: Exception: 'TokenizersWrapper' object has no attribute 'bos_token_id'

### ISSUE THREAD

**Issue by DJLegends1011** [MinusZoneAI/ComfyUI-TrainTools-MZ#21]
currently have an issue when CMD open and I have this

override steps. steps for 100 epochs is / 指定エポックまでのステップ数: 1000
running training / 学習開始
  num train images * repeats / 学習画像の数×繰り返し回数: 10
  num reg images / 正則化画像の数: 0
  num batches per epoch / 1epochのバッチ数: 10
  num epochs / epoch数: 100
  batch size per device / バッチサイズ: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 1000
steps:   0%|                                                                                  | 0/1000 [00:00<?, ?it/s]sample_images: height = 1024, width = 1024
2025-02-24 04:43:55 INFO     prompt: score_9, Echosaber, 1girl                                        train_util.py:5284
                    INFO     negative_prompt:                                                         train_util.py:5285
                    INFO     height: 1024                                                             train_util.py:5286
                    INFO     width: 1024                                                              train_util.py:5287
                    INFO     sample_steps: 20                                                         train_util.py:5288
                    INFO     scale: 5.0                                                               train_util.py:5289
                    INFO     sample_sampler: euler_a                                                  train_util.py:5290
                    INFO     seed: 0                                                                  train_util.py:5292
Exception: 'TokenizersWrapper' object has no attribute 'bos_token_id'

**Comment by DJLegends1011** [comment]
wierd this is that Huan  template works but not SD1.5 nor XL
